BookingsApi/BookingsApi.IntegrationTests/Api/V1/JudiciaryParticipants/JudiciaryParticipantApiTest.cs

```diff
@@ -23,5 +24,25 @@ protected void AssertEventsPublishedForNewJudiciaryParticipants(Hearing hearing,
                 .Should()
                 .BeEquivalentTo(new ParticipantsAddedIntegrationEvent(hearing, judiciaryParticipants));
         }
+        
+        protected void AssertEventsPublishedForNewJudiciaryParticipantsNotification(Hearing hearing, IEnumerable<JudiciaryParticipant> judiciaryParticipants)
```

Comment:
a few points on this method:

the `judiciaryParticipants` argument isn't used.

I would suggest passing in the new judiciary participants from your test to this method.

we can then replace the below

```
var participantAddedMessage = messages.ToList().Find(x => x.IntegrationEvent is HearingNotificationIntegrationEvent);
            participantAddedMessage.Should().NotBeNull();
            
            var participantMessages = messages
                .Where(x => x.IntegrationEvent is HearingNotificationIntegrationEvent)
                .Select(x => x.IntegrationEvent as HearingNotificationIntegrationEvent)
                .Where(x => x.HearingConfirmationForParticipant.HearingId == hearing.Id)
                .ToList();
            
            
            participantMessages.Count.Should().Be(1);
```

with the following:

```
var notificationMessages = messages
       .Where(x => x.IntegrationEvent is HearingNotificationIntegrationEvent)
       .ToList();
notificationMessages.Should().NotBeNull();
notificationMessages.Count.Should().Be(judiciaryParticipants.Count());
```

this way we're not using a hardcoded count.

Reply:
test refactored